src/utils/useLocationUpdateEffect.ts

```diff
@@ -33,6 +43,7 @@ export const updateLocationData = (
 };
 
 // get Profile from the firebase
+// @debt rename this trackRoomEntered
```

Comment:
curiosity: did you hold off on this rename to keep the PR from exploding across many files?

Reply:
Partially that, and partially because I technically already did it in https://github.com/sparkletown/sparkle/pull/840

https://github.com/sparkletown/sparkle/pull/840/files?file-filters%5B%5D=.ts&hide-deleted-files=true#diff-dc94dd179cbd4f837ed8252c9624d3c3fca4b5e6caefdb499d77616685870dcbR36